.gitignore

```diff
@@ -12,3 +12,5 @@
 __pycache__/
 .pytest_cache
 .tox
+Vagrantfile
```

Comment:
not needed, we are not testing in vagrant :)

Reply:
Exactly ! I was doing test localy on vagrant and kept the .gitignore with that for a while.. I guess I commited that by error :)